src/AccessibilityInsights.SharedUx/FileIssue/IssueReporterManager.cs

```diff
@@ -66,7 +67,7 @@ private IssueReporterManager()
                 catch (ArgumentException ex)
                 {
                     // Fail silently in case of dups.
-                    Console.WriteLine("Found duplicate extensions " + ex.StackTrace);
+                    Console.WriteLine("Found duplicate extensions / Extension failed to restore " + ex.StackTrace);
```

Comment:
What's the final plan for reporting this case? Maybe calling ReportException?

Reply:
Added a call to reportexception through Logger as discussed offline.